core/lib/components/balena/sdk.js

```diff
@@ -175,7 +175,7 @@ module.exports = class BalenaSDK {
 		const repo = await this.balena.models.application
 			.get(application)
 			.get('slug');
-		const balenaConfig = await this.balena.models.config.getAll();
+		const balenaConfig = await this.balena.models.config.l();
```

Comment:
What does `.l()` do, or is this a typo?

Reply:
It seems my bulk changes have had this issue creep in. Thanks for the eagle eye reviewing